notebooks/examples/analog_sensitivity_LeNet5.ipynb

```diff
@@ -0,0 +1,1483 @@
+      "Accuracy of the digital model: 96.02%\n"
+     ]
+    }
+   ],
+   "source": [
+    "dmodel, _, _, digital_accuracy = test_evaluation(validation_data, dmodel, criterion)\n",
+    "print(f\"Accuracy of the digital model: {digital_accuracy:.2f}%\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14ceee0f",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Experiment 1: Analog Inference\n",
+    "\n",
+    "As a preliminary experiment, we perform \"analog\" inference of the previously trained digital LeNet with a floating point synaptic device model, while ignoring the non-idealities of peripheral circuits. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "d0b885ff",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AnalogSequential(\n",
+      "  (0): AnalogConv2d(1, 16, kernel_size=(5, 5), stride=(1, 1), digital bias)\n",
+      "  (1): ReLU()\n",
+      "  (2): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)\n",
+      "  (3): AnalogConv2d(16, 32, kernel_size=(5, 5), stride=(1, 1), digital bias)\n",
+      "  (4): ReLU()\n",
+      "  (5): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)\n",
+      "  (6): ReLU()\n",
+      "  (7): Flatten(start_dim=1, end_dim=-1)\n",
+      "  (8): AnalogLinear(in_features=512, out_features=128, bias=True, digital bias)\n",
+      "  (9): ReLU()\n",
+      "  (10): AnalogLinear(in_features=128, out_features=10, bias=True, digital bias)\n",
+      "  (11): LogSoftmax(dim=1)\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "rpu_config = FloatingPointRPUConfig(device=FloatingPointDevice())\n",
+    "amodel = convert_to_analog(dmodel, rpu_config)\n",
+    "print(amodel)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "34425850",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy of the analog model: 96.02%\n",
+      "Accuracy degradation: 0.00%\n"
+     ]
+    }
+   ],
+   "source": [
+    "amodel, _, _, accuracy = test_evaluation(validation_data, amodel, criterion)\n",
+    "print(f\"Accuracy of the analog model: {accuracy:.2f}%\")\n",
+    "print(f\"Accuracy degradation: {digital_accuracy-accuracy:.2f}%\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdf3ea57",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Experiment 2: Parameters of Analog Inference\n",
+    "\n",
+    "In this experiment, we introduce a few of the non-idealities involved in analog inference. Experimenting with different settings for these factors shows the clear impact of analog inference on accuracy."
```

Comment:
You could add that for analog inference, only non-idealities in the forward pass will be affecting the accuracy, as backward (and update) is considered perfect and ideal. We should always use InferenceRPUConfig for analog inference chips.